rails/app/models/portal/run.rb

```diff
@@ -0,0 +1,10 @@
+class Portal::Run < ApplicationRecord
+
+    self.table_name = :portal_runs
+
+    # Associations
+    belongs_to :portal_learner, class_name: "Portal::Learner", foreign_key: "learner_id"
```

Comment:
I think you need to either add an `:inverse_of => :portal_run` here or remove the inverse of in the new portal learner has_many declaration.

Reply:
I'm not totally sure, but I think the current code is right. Based on the docs, the `inverse_of` is supposed to go on the `has_many`: https://guides.rubyonrails.org/association_basics.html#bi-directional-associations